MeteoSwiss-APN/dawn
# ICON stencil from sir

## Technical Description

Rewrites the ICON laplacian stencil in SIR (with dawn4py) and adds it to dawn4py tests.

### Resolves / Enhances

resolves #692 

### Testing

Coupling with driver code: https://github.com/mroethlin/AtlasUtilities/pull/1
To verify that output didn't change:
- build repo (follow instructions on root README)
- `python3 convergence_plot.py ./atlasIconLaplaceDriver`
- `python3 convergence_plot.py ./mylibIconLaplaceDriver`
- repeat previous for master (using a different build folder)
- diff *.txt files with same name
